src/widlprocess.ts

```diff
@@ -33,7 +33,10 @@ export function convert(text: string, commentMap: Record<string, string>) {
             browser.namespaces!.push(convertNamespace(rootType, commentMap));
         }
         else if (rootType.type === "callback interface") {
-            browser["callback-interfaces"]!.interface[rootType.name] = convertInterface(rootType, commentMap);
+            browser["callback-interfaces"]!.interface[rootType.name] = convertInterface(
```

Comment:
I think it's better to do this in `convertInterface` function signature itself. Probably `InterfaceType | CallbackInterfaceType`.

Reply:
Filed https://github.com/DefinitelyTyped/DefinitelyTyped/pull/47252, this won't be needed if it's accepted.